internal/services/compute/virtual_machine_scale_set_import.go

```diff
@@ -18,30 +18,14 @@ func importOrchestratedVirtualMachineScaleSet(ctx context.Context, d *pluginsdk.
 
 	client := meta.(*clients.Client).Compute.VMScaleSetClient
 	// Upgrading to the 2021-07-01 exposed a new expand parameter in the GET method
-	vm, err := client.Get(ctx, id.ResourceGroup, id.Name, compute.ExpandTypesForGetVMScaleSetsUserData)
+	_, err = client.Get(ctx, id.ResourceGroup, id.Name, compute.ExpandTypesForGetVMScaleSetsUserData)
 	if err != nil {
 		return []*pluginsdk.ResourceData{}, fmt.Errorf("retrieving Virtual Machine Scale Set %q (Resource Group %q): %+v", id.Name, id.ResourceGroup, err)
 	}
 
-	if err := assertOrchestratedVirtualMachineScaleSet(vm); err != nil {
-		return []*pluginsdk.ResourceData{}, fmt.Errorf("importing Virtual Machine Scale Set Orchestrator VM %q (Resource Group %q): %+v", id.Name, id.ResourceGroup, err)
-	}
```

Comment:
we'll need this to confirm this isn't a non-orchestrated VMSS, albeit we need to update this to check the type/profile?

Reply:
Still trying to figure out exactly how to work that out at the moment as the flex has added a wrinkle into this check... I am open to ideas if you might have a way to derive what "type" of VM is attempting to be imported.